### DISCUSSION TITLE
Kandidaat als favoriet aanduiden

### DISCUSSION THREAD

**Discussion by IngeVerding** [Nelos-VZW/dives-2#25] [Q&A]
Wat is het nut een kandidaat als favoriet aan te duiden? 
Het is niet dat je vandaar een duik kan loggen of zo

PS ik kan enkel testen als 3*D of AI, dus misschien is het wel nuttig voor instructeurs of duikschoolverantwoordelijken

**Comment by ndsmyter** [comment]
Kandidaten zijn inderdaad bedoeld voor instructeurs.
Als ik voor mezelf spreek in DIVES 1.0, zijn er heel wat mensen die mij als coach aangeduid hebben over de jaren. Dus mijn lijstje van mensen is heel erg groot, maar eigenlijk zijn er maar een paar mensen waarin ik echt geïnteresseerd ben. Dus dat was onze uitgangspositie.
We willen gewoon toelaten dat een instructeur bepaalde mensen snel kan gaan opvolgen, typisch de mensen die bij hem/haar een opleiding volgen.

PS: Als 3*D of AI had je deze knop eigenlijk niet mogen zien. Dat is ondertussen wel opgelost :-)
